src/api/controllers/articleController.js

```diff
@@ -0,0 +1,71 @@
+import _ from 'lodash';
+import models, { sequelize } from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import sendError from '../../helpers/error.sender';
+import calculateRatingsPercentage from '../../helpers/calculate.ratings.percentages';
+import rawQueries from '../../helpers/constants/raw.queries';
+import generateRatings from '../../helpers/generate.ratings';
+import errorMessages from '../../helpers/constants/error.messages';
+
+const { Rating } = models;
+
+/**
+ * containing aal controllers of the ratings process
```

Comment:
@nkpremices `all`

Reply:
Oh!!!! I'm sorry about that.
Let me correct it..